syz-manager/rpc.go

```diff
@@ -94,15 +96,21 @@ func (serv *RPCServer) Connect(a *rpctype.ConnectArgs, r *rpctype.ConnectRes) er
 	if err != nil {
 		return err
 	}
+
 	serv.coverFilter = coverFilter
 	serv.modules = a.Modules
 
+	if serv.canonicalModules == nil {
+		serv.canonicalModules = cover.NewCanonicalizer(a.Modules)
+	}
+
 	serv.mu.Lock()
 	defer serv.mu.Unlock()
 
 	f := &Fuzzer{
 		name:        a.Name,
 		machineInfo: a.MachineInfo,
+		instModules: serv.canonicalModules.NewInstance(a.Modules),
 	}
 	serv.fuzzers[a.Name] = f
```

Comment:
Just in case: there's also the `CoverFilterBitmap` field, it might be also affected by these PC transformations.

Reply:
That's a very good point. Currently there's only one coverage filter per manager, but we'd need one per fuzzer to account for modules. As it's a somewhat larger change, I'll cover this in a separate CL if that's alright.

Reply:
> As it's a somewhat larger change, I'll cover this in a separate CL if that's alright.

Sure. For now you could e.g. modify `Manager.fuzzerConnect` to do `log.Fatalf` if there are modules and a coverage filter.